packages/tocco-ui/src/EditableValue/typeEditors/RemoteSelect.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Select from 'react-select'
+
+class RemoteSelect extends React.Component {
+  loadOptions = searchTerm => {
+    return this.props.options.fetchOptions(searchTerm).then(res => ({options: res, complete: false}))
+  }
```

Comment:
`{...}` and `return` not needed for this one-liner: `searchTerm => this.props...`